builtin/providers/aws/resource_aws_route.go

```diff
@@ -152,7 +155,27 @@ func resourceAwsRouteCreate(d *schema.ResourceData, meta interface{}) error {
 	log.Printf("[DEBUG] Route create config: %s", createOpts)
 
 	// Create the route
-	_, err := conn.CreateRoute(createOpts)
+	var out *ec2.CreateRouteOutput
```

Comment:
it doesn't look like we use `out` here; it's used in `conn.CreateRoute` but then never after that, safe to remove?